docs/DeveloperGuide.md

```diff
@@ -307,26 +306,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2a1. TAB displays warning message to user (that there are no students yet)
 
       Use case ends
-* 2b. TAB detects that an identical lab tag already exists.
-    * 2b1. TAB displays error message (that tag already exists)
+* 2b. TAB detects that an identical lab already exists.
+    * 2b1. TAB displays error message (that lab already exists)
 
       Use case ends
 
 **Use case: Filter Students**
 
 **MSS**
 
-1.  TA requests to filter students by a tag 
+1.  TA requests to filter students by a specific lab 
 2.  TAB displays list of students based on filter criteria
 
     Use case ends
 
 **Extensions**
 
-* 1a. TA provides non-existent tag
+* 1a. TA provides non-existent lab
   
     * 1a1. TAB shows error message
-    * 1a2. TAB requests for valid tag
+    * 1a2. TAB requests for valid lab
```

Comment:
Maybe just end the Use Case here?

Reply:
Valid point. I think both ways work here. Which do you think is more natural? @channne

Reply:
I think we could also modify the use case to include scenarios where the TA stops providing additional input. Maybe we can save these ideas and decide which one is best to implement after we understand the product better?

Reply:
Alright since no one is rejecting it, for now, I guess we can merge this?

Reply:
agreed